src/configeditor/EditBoundingBoxes.tsx

```diff
@@ -0,0 +1,208 @@
+  Button,
+  VStack,
+  FormControl,
+  FormLabel,
+  Text,
+  HStack,
+  FormErrorMessage,
+} from "@chakra-ui/react";
+import { Config } from "../types/config";
+import { BoundingBox } from "../util/bounds";
+import { ErrorObject, useConfigEdit } from "./hooks";
+
+interface LatLngInputProps {
+  title: string;
+  field: string;
+  value: string;
+  error: boolean;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+interface EditSingleBoundingBoxesProps {
+  type: string;
+  bounds: BoundingBox | null;
+  values: any;
+  setValues: any;
+  error: boolean;
+}
+
+function LatLngInput(props: LatLngInputProps) {
+  return (
+    <FormControl isInvalid={props.error}>
+      <FormLabel htmlFor={props.field}>{props.title}</FormLabel>
+      <Input
+        id={props.field}
+        type="number"
+        variant="outline"
+        value={props.value}
+        onChange={props.onChange}
+      />
+    </FormControl>
+  )
+}
+
+function EditSingleBoundingBoxes(props: EditSingleBoundingBoxesProps) {
+  const { type, bounds, values, setValues, error } = props;
+
+  const onChange = (value: string, field: "westmost" | "southmost" | "eastmost" | "northmost") => {
+    if (!bounds) {
+      const newBounds = (
+        field === "westmost" ? new BoundingBox(value, "", "", "") :
+        field === "southmost" ? new BoundingBox("", value, "", "") :
+        field === "eastmost" ? new BoundingBox("", "", value, "") :
+        new BoundingBox("", "", "", value)
+      );
+      setValues({
+        ...values,
+        boundingBoxes: {
+          ...values.boundingBoxes,
+          [type]: newBounds.toConfigBbox()
+        }
+      });
+    } else {
+      bounds[field] = value;
+      setValues({
+        ...values,
+        boundingBoxes: {
+          ...values.boundingBoxes,
+          [type]: bounds.toConfigBbox()
+        }
+      });
+    };
+  };
+
+  return (
+    <HStack align="start">
+      <LatLngInput
+        title="West"
+        field="westmost"
+        value={bounds ? bounds.westmost : ""}
+        onChange={e => onChange(e.target.value, "westmost")}
+        error={error}
+      />
+      <LatLngInput
+        title="South"
+        field="southmost"
+        value={bounds ? bounds.southmost : ""}
+        onChange={e => onChange(e.target.value, "southmost")}
+        error={error}
+      />
+      <LatLngInput
+        title="East"
+        field="eastmost"
+        value={bounds ? bounds.eastmost : ""}
+        onChange={e => onChange(e.target.value, "eastmost")}
+        error={error}
+      />
+      <LatLngInput
+        title="North"
+        field="northmost"
+        value={bounds ? bounds.northmost : ""}
+        onChange={e => onChange(e.target.value, "northmost")}
+        error={error}
+      />
+      <button
+        key={type}
+        title={"Clear input"}
+        onClick={() => setValues({
+          ...values,
+          boundingBoxes: {
+            ...values.boundingBoxes,
+            [type]: null
```

Comment:
Instead of `null`, it should be `undefined`, I think -- it's an optional field, that should be undefined if it's missing.